src/main/java/it/gov/pagopa/payhub/pdnd/service/PdndClient.java

```diff
@@ -0,0 +1,12 @@
+package it.gov.pagopa.payhub.pdnd.service;
```

Comment:
let's split the service layer, which should contains just the business logic, with the integration layer

```suggestion
package it.gov.pagopa.payhub.pdnd.connector.pdnd.client;
```